samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/state_binding.c

```diff
@@ -224,6 +224,26 @@ void state_binding(u8_t light, u8_t temp)
 	readjust_lightness();
 }
 
+void init_lightness_target_values(void)
+{
+	target_lightness = lightness;
+
+	if (target_lightness) {
+		gen_onoff_srv_root_user_data.target_onoff = STATE_ON;
+	} else {
+		gen_onoff_srv_root_user_data.target_onoff = STATE_OFF;
+	}
+
+	gen_level_srv_root_user_data.target_level = target_lightness + INT16_MIN;
+
+	light_lightness_srv_user_data.target_actual = target_lightness;
+
+	light_lightness_srv_user_data.target_linear =
+		actual_to_linear(target_lightness);
+
+	light_ctl_srv_user_data.target_lightness = target_lightness;
```

Comment:
As I mentioned earlier, all of the initialisation done by this function should be possible to do at build time rather than runtime. Only way this would make a little bit sense is if you called this code *after* loading settings, but that's not what the current code is doing (it's calling settings_load() from bt_ready, which is called after these init functions)